src/dist.config.ts

```diff
@@ -77,7 +77,10 @@ function webpackConfig(args: any): webpack.Configuration {
 					? manifest.icons.map((icon) => ({ ...icon, ios: true }))
 					: manifest.icons
 			}),
-		new webpack.BannerPlugin(banner),
+		new webpack.BannerPlugin({
+			banner,
+			test: /^(.(?!.*\.css$))*$/i
```

Comment:
have we considered just writing a positive regex for `.js` files rather than a more complicated one to exclude the css? I can't think of anything other than `.js` to apply the copyright to.

Reply:
That would certainly make this regex cleaner. I didn't want to have more of an impact than I intended but I can happily clean this up and just do js files! 👍